app/ide-desktop/lib/dashboard/src/components/AriaComponents/Dialog/types.ts

```diff
@@ -5,6 +5,7 @@ import type * as aria from '#/components/aria'
 export interface DialogProps extends aria.DialogProps {
   readonly title?: string
   readonly isDismissable?: boolean
+  readonly isOpen?: boolean
```

Comment:
@somebody1234 we have `modalProps` for that

Reply:
I guess we can flatten `modalProps` because it seems like it's not clear that we can control the open/close using it

Reply:
just tried, seems like `className` conflicts with `aria.DialogProps` which this interface also extends. but it seems like it's not being used anyway so i've just removed it